Traceback (most recent call last):
  File "C:\Anaconda\lib\site-packages\jupyter_cache\executors\utils.py", line 58, in single_nb_execution
    executenb(
  File "C:\Anaconda\lib\site-packages\nbclient\client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "C:\Anaconda\lib\site-packages\nbclient\util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "C:\Anaconda\lib\site-packages\nbclient\util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "C:\Anaconda\lib\asyncio\base_events.py", line 647, in run_until_complete
    return future.result()
  File "C:\Anaconda\lib\site-packages\nbclient\client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "C:\Anaconda\lib\site-packages\nbclient\client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "C:\Anaconda\lib\site-packages\nbclient\client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
import geopandas as gpd
from geopandas import GeoSeries
from shapely.geometry import Point, LineString
import folium 
from folium import Marker, GeoJson
from folium.plugins import MarkerCluster, HeatMap
------------------

[1;31m---------------------------------------------------------------------------[0m
[1;31mModuleNotFoundError[0m                       Traceback (most recent call last)
[1;32m~\AppData\Local\Temp\ipykernel_4092\3230868528.py[0m in [0;36m<module>[1;34m[0m
[0;32m      2[0m [1;32mfrom[0m [0mgeopandas[0m [1;32mimport[0m [0mGeoSeries[0m[1;33m[0m[1;33m[0m[0m
[0;32m      3[0m [1;32mfrom[0m [0mshapely[0m[1;33m.[0m[0mgeometry[0m [1;32mimport[0m [0mPoint[0m[1;33m,[0m [0mLineString[0m[1;33m[0m[1;33m[0m[0m
[1;32m----> 4[1;33m [1;32mimport[0m [0mfolium[0m[1;33m[0m[1;33m[0m[0m
[0m[0;32m      5[0m [1;32mfrom[0m [0mfolium[0m [1;32mimport[0m [0mMarker[0m[1;33m,[0m [0mGeoJson[0m[1;33m[0m[1;33m[0m[0m
[0;32m      6[0m [1;32mfrom[0m [0mfolium[0m[1;33m.[0m[0mplugins[0m [1;32mimport[0m [0mMarkerCluster[0m[1;33m,[0m [0mHeatMap[0m[1;33m[0m[1;33m[0m[0m

[1;31mModuleNotFoundError[0m: No module named 'folium'
ModuleNotFoundError: No module named 'folium'

